src/vm/types/signatures.rs

```diff
@@ -56,8 +56,8 @@ pub const BUFF_20: TypeSignature = BufferType(BufferLength(20));
 
 #[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
 pub struct ListTypeData {
-    max_len: u32,
-    entry_type: Box<TypeSignature>
+    pub max_len: u32,
```

Comment:
It would be better for these to remain private, and add or use accessor functions. By making these public, you can circument the struct's constructor, which is responsible for performing max value size checks.

Reply:
Addressed with https://github.com/blockstack/blockstack-core/pull/1139/commits/bdcc16d0ba103dba696e9a71ab71432329ccbc4e